be/src/exec/assert_num_rows_node.cpp

```diff
@@ -45,6 +45,7 @@ Status AssertNumRowsNode::prepare(RuntimeState* state) {
 Status AssertNumRowsNode::open(RuntimeState* state) {
     SCOPED_TIMER(_runtime_profile->total_time_counter());
     RETURN_IF_ERROR(ExecNode::open(state));
+    RETURN_IF_ERROR(child(0)->open(state));
```

Comment:
Would better add a comment or this issue link to let other people know why and when to call child(0)->open method. Thanks.

Reply:
Added. Actually, I think most of expr node should call the child.open in open function. This is my mistake.